functions/executor/call.py

```diff
@@ -146,36 +152,40 @@ def _exec_dag_function_causal(pusher_cache, kvs, triggers, function, schedule):
     fname = schedule.target_function
     fargs = list(schedule.arguments[fname].args)
 
-    address_to_versioned_key_list_map = {}
-    dependency = {}
+    versioned_key_locations = {}
+    dependencies = {}
 
     for trname in schedule.triggers:
         trigger = triggers[trname]
         fargs += list(trigger.arguments.args)
         # combine address_versioned_key_list pairs
-        for addr in trigger.address_to_versioned_key_list_map:
-            address_to_versioned_key_list_map[addr] = list(trigger.address_to_versioned_key_list_map[addr].versioned_keys)
-        # combine dependency from previous func
-        for dep in trigger.dependency:
-            if dep.key in dependency:
-                dependency[dep.key] = _merge_vector_clock(dependency[dep.key], dep.vector_clock)
+        for addr in trigger.versioned_key_locations:
+            versioned_key_locations[addr] = \
+            list(trigger.versioned_key_locations[addr].versioned_keys)
+        # combine dependencies from previous func
+        for dep in trigger.dependencies:
+            if dep.key in dependencies:
+                dependencies[dep.key] = \
+                _merge_vector_clock(dependencies[dep.key], dep.vector_clock)
             else:
-                dependency[dep.key] = dep.vector_clock
+                dependencies[dep.key] = dep.vector_clock
 
-    logging.info('Executing function %s for DAG %s (ID %d) in causal consistency.' %
-            (schedule.dag.name, fname, trigger.id))
+    logging.info('Executing function %s for DAG %s (ID %d) in '
+        'causal consistency.' % (schedule.dag.name, fname, trigger.id))
```

Comment:
This is not valid Python syntax. If you want to concat the strings, you need to add a +.